docs/development/models/member.md

```diff
@@ -17,6 +17,78 @@ lang: php
 
 [TOC]
 
+## Properties
+
+### Required:
+#### `role_id` 
+#### `username`
+#### `email`
+#### `password`
+
+### Optional:
+#### `member_id`
+#### `group_id`
```

Comment:
I believe `group_id` property needs not be in the docks. Or at least marked as deprecated. It is present for the sake of making legacy code work, but when writing new code, `role_id` should always be used